README.md

```diff
@@ -494,7 +494,7 @@ interfering with the transmit thread.
 ## Portability
 
 The code runs well on Linux, Windows, and Mac OS X. All the important bits are
-in standard C (C90). It therefore compiles on Visual Studio with Microsoft's
+in standard C (C90). It, therefore, compiles on Visual Studio with Microsoft's

```

Comment:
This change is a bit pedantic. The commas aren't needed, and make it more awkward to read.